download/linux.md

```diff
@@ -2,8 +2,21 @@
 title: Install MonoDevelop on Linux
 ---
 
-Install MonoDevelop on Linux
-============================
+Install MonoDevelop preview via FlatPak
+=======================================
+
+[FlatPak](http://flatpak.org/) is a new packaging format for Linux, which allows the same applications to run in a sandboxed environment, across any modern distribution.
+
+We are offering a preview release of MonoDevelop as a FlatPak bundle, which does not require Mono to be installed.
+
+[Download the preview here]({{ site.data.latestrelease.monodevelop_linux_url }}).
+
+Please follow step 1 and 2 from [here](http://flatpak.org/#users) as appropriate, and follow step 3 using the runtime `org.freedesktop.Platform`
```

Comment:
I think it makes more sense to link to the [Getting FlatPak](http://flatpak.org/getting.html) page and then just show the other instructions on this page directly, so I don't have to think about what step 1 and 2 are, and which part in step 3 I need to change.